rollup/internal/controller/watcher/chunk_proposer.go

```diff
@@ -184,14 +190,31 @@ func (p *ChunkProposer) updateChunkInfoInDB(chunk *types.Chunk) error {
 	return err
 }
 
+// blocksUntilFork returns the number of blocks until the next fork
+// returns 0 if there is no fork scheduled for the future
+func blocksUntilFork(blockHeight uint64, forkHeights []uint64) uint64 {
```

Comment:
```suggestion
func (p *ChunkProposer) blocksUntilFork(blockHeight uint64, forkHeights []uint64) uint64 {
```

Reply:
I will move it to `forks`package, if that makes sense to you as well.

Reply:
yeah. make sense